test/bigtest/tests/receiving-list-test.js

```diff
@@ -85,5 +88,132 @@ describe('Receiving', () => {
         expect(orderDetailsPage.$root).to.exist;
       });
     });
+
+    describe('check one line and enable Receive Pieces button', () => {
+      beforeEach(async () => {
+        await receivingPage.receivingList(0).click();
+      });
+
+      it('Receive pieces button is enabled', () => {
+        expect(receivingPage.receivePiecesButton.isDisabled).to.be.false;
+      });
+    });
+
+    describe('check all lines and enable Receive Pieces button', () => {
+      beforeEach(async () => {
+        await receivingPage.checkbox.click();
+      });
+
+      it('Receive pieces is enabled', () => {
+        expect(receivingPage.receivePiecesButton.isDisabled).to.be.false;
```

Comment:
I think it might be better to have a separate interactor with modal instead of writing "god" interactor containing all the logic.

Reply:
Done